SPEC.md

```diff
@@ -201,25 +201,170 @@ Note that logging mechanisms other than stdout and stderr are not required by th
 ### Isolators
 
 Isolators enforce resource constraints rather than namespacing.
-Isolators may be applied to individual applications, to whole containers, or to both.
+Isolators may be scoped to individual applications, to whole containers, or to both.
```

Comment:
I think "notify the user" is ok but does somewhat conjure the image of a human in front of a keyboard (e.g. Rocket). An executor might be a component in a larger system. Something like "made available"? Related, should isolation information (ignored/enforced/modified) be made available to the container by way of the metadata server?